README.md

```diff
@@ -1 +1,3 @@
-# spark-etl
\ No newline at end of file
+# spark-etl
+
+[![Join the chat at https://gitter.im/vngrs/spark-etl](https://badges.gitter.im/vngrs/spark-etl.svg)](https://gitter.im/vngrs/spark-etl?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
@gitter-badger Do we need these utm query strings? If we don't please remove them.